spec/support/tower_data_helper.rb

```diff
@@ -0,0 +1,21 @@
+module Spec
```

Comment:
Similarly here, I think we don't need this to achieve what this PR intends to. In #93 you can simply load from either  a given yml file or the default one.